addons/mail/static/src/js/owl/component/thread.js

```diff
@@ -39,6 +44,10 @@ class Thread extends owl.store.ConnectedComponent {
         ) {
             this._loadThreadCache();
         }
+        if(this._messageListJustMounted)
+        {
+            this._handleInitialScroll();
```

Comment:
I prefer `this._isMessageListJustMounted = false;` right there, then `_handleMessageListScrollOnMount()`:

```js
if (this._isMessageListJustMounted) {
   this._isMessageListJustMounted = false;
   this._handleMessageListScrollOnMount();
}
```